api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4691,7 +4690,6 @@ public void putObject(String bucketName, String objectName, InputStream stream,
    * @throws InternalException           upon internal library error
    * @throws InvalidArgumentException    upon invalid value is passed to a method.
    * @throws InvalidResponseException    upon a non-xml response from server
-   * @see #putObject(String bucketName, String objectName, String fileName)
```

Comment:
Why do we remove this?

Reply:
Here we were referring to the overloaded function which is already deprecated.